prow/jobs/kyma/components/kyma-components-static-checks.yaml

```diff
@@ -285,7 +285,7 @@ presubmits: # runs on PRs
         prow.k8s.io/pubsub.project: "sap-kyma-prow"
         prow.k8s.io/pubsub.runID: "pull-event-publisher-proxy-unit-test"
         prow.k8s.io/pubsub.topic: "prowjobs"
-      run_if_changed: '^components/event-publisher-proxy/.*\.go$'
+      run_if_changed: 'components/event-publisher-proxy/(?!.*.md)(?!.*.gitignore).*'
```

Comment:
I think we can retain the `^` at the beginning right?

Reply:
Done